marginaleffects/datagrid.py

```diff
@@ -11,6 +11,7 @@ def datagrid(
     grid_type="mean_or_mode",
     FUN_numeric=lambda x: x.mean(),
     FUN_other=lambda x: x.mode()[0],  # mode can return multiple values
+    FUN_categorical=lambda x: x.unique(),
```

Comment:
We probably need to add something similar on line 66

Reply:
you are right, we don't need it. I was aiming to replicate the code in R in case it would be useful